package.json

```diff
@@ -6,11 +6,11 @@
   "scripts": {
     "lint": "standard",
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
-    "build": "yarn upgrade && browserify client/sync.js -o bundles/bundle.js",
-    "browsertest": "yarn start-test; browserify test/client/*.js test/*.js | tape-run; yarn stop-test",
-    "browsertest-client": "yarn start-test; browserify test/client/*.js | tape-run --browser chrome; yarn stop-test",
+    "build": "yarn upgrade && browserify client/sync.js | uglifyjs - --mangle > bundles/bundle.js",
```

Comment:
i think we should leave out mangling since it doesn't seem to save much. that would make the tests run over the same thing that is shipping